back-end/routes/stitchRoutes.js

```diff
@@ -10,14 +10,20 @@ router.get('/', async (req, res) => {
     res.json(stitches)
 })
 
-router.get('/user', async (req, res) => {
-    const { username } = req.body
+router.get('/:username', async (req, res) => {
```

Comment:
> I might've miss this, but just want to make sure `user` was renamed to `username`, right?

It wasn't renamed I am just now getting the username from the request call rather than the body!